### PR TITLE
Okta spring security split

### PR DESCRIPTION
**Pull request by dogeared**
This blog post has the following features:

- [X] A GitHub Repository with a polished README
- [ ] A title that's approved by Dev Marketing
- [ ] A URL that's approved by Dev Marketing
- [X] Rendered locally and confirmed that no Markdown typos exist

### CODE REVIEW

**Review by mraible — APPROVED** [review]

File: _source/_posts/2020-10-08-okta-spring-security-feature-flags.md [hunk 1]
```diff
@@ -0,0 +1,442 @@
+---
```
Comment (mraible) [inline _source/_posts/2020-10-08-okta-spring-security-feature-flags.md:1]: Change the date to tomorrow by renaming the file.

File: _source/_posts/2020-10-08-okta-spring-security-feature-flags.md [hunk 2]
```diff
@@ -0,0 +1,442 @@
...
+
+In the example for this post, you integrate a split **treatment** such that a select group of users sees a new 'beta' interface for your app while ordinary users see the current production interface.
+
+When our imaginary beta testing is complete, you can enable the new interface for all users.
+
+Best of all, you could repeat this process, creating new beta experiences to be tested by your group and eventually making that generally available.
+
+Getting setup with a free developer account for Split is as easy as 1, 2, 3:
+
+1. Go to: https://split.io, click: Free Account
```
Comment (mraible) [inline _source/_posts/2020-10-08-okta-spring-security-feature-flags.md:244]:
```suggestion
1. Go to: <https://split.io>, click: Free Account
```

File: _source/_posts/2020-10-08-okta-spring-security-feature-flags.md [hunk 3]
```diff
@@ -0,0 +1,442 @@
...
+You could do this over and over again and never touch the controller code. The only thing that's changing are the templates and the settings in split.
+
+This approach also lends itself to changing who is in your beta test program without having to change your code.
+
+In a real application, you'd be working with a database or an Identity Management system where you could add and remove users from the **BETA_TESTER** group. With Okta, you can easily manage who belongs to the group from the admin console or via the Okta Management API. Those users would always see the latest and greatest beta while ordinary users would see only the current release.
+
+## Learn More About Building Secure Applications
+
+I hope you've seen how useful it can be to set up different experiences for different users using Okta, Split and the native functionality built into Spring Security.
+
```
Comment (mraible) [inline _source/_posts/2020-10-08-okta-spring-security-feature-flags.md:432]: Is there a GitHub repo you can link to?

**Review by mraible — APPROVED** [review]

File: _source/_posts/2020-10-08-okta-spring-security-feature-flags.md (hunk 1 above)
Comment (dogeared) [reply _source/_posts/2020-10-08-okta-spring-security-feature-flags.md]: ack

File: _source/_posts/2020-10-08-okta-spring-security-feature-flags.md (hunk 3 above)
Comment (dogeared) [reply _source/_posts/2020-10-08-okta-spring-security-feature-flags.md]: yep. I pushed up a new version with that link.

**Pull request merged**